app/helpers/avatar_helper.rb

```diff
@@ -0,0 +1,17 @@
+module AvatarHelper
+  def avatar_path(object, options = {})
+    size = options[:size] || 180
+    default_image = options[:default] || "mp"
+    base_url =  "https://secure.gravatar.com/avatar"
+    base_url_params = "?s=#{size}&d=#{default_image}"
+
+    if object.respond_to?(:avatar) && object.avatar.attached? && object.avatar.variable?
+      object.avatar.variant(resize_to_fill: [size, size, { gravity: 'Center' }])
+    elsif object.respond_to?(:email) && object.email
+      gravatar_id = Digest::MD5::hexdigest(object.email.downcase)
+      "#{base_url}/#{gravatar_id}#{base_url_params}"
+    else
+      "#{base_url}/00000000000000000000000000000000#{base_url_params}"
```

Comment:
wow what are the 00000 ? is that an image?

Reply:
this is to simulate gravatar's email hash which is 32char long
<img width="650" alt="image" src="https://user-images.githubusercontent.com/70934030/123516815-5b23f180-d6e1-11eb-91c7-fd07212129c5.png">
<img width="680" alt="image" src="https://user-images.githubusercontent.com/70934030/123516834-83abeb80-d6e1-11eb-9304-2c4c826af04b.png">